src/Ripemd160.mo

```diff
@@ -57,9 +57,9 @@ module {
 
     private func round(
       a : Nat32,
-      b : Nat32,
+      _b : Nat32,
```

Comment:
Why is `b` changed to `_b`? (Same question for `d`)

Reply:
it's unused 🤷